immuneML/ml_methods/DeepRC.py

```diff
@@ -23,6 +23,8 @@ class DeepRC(MLMethod):
     This classifier uses the DeepRC method for repertoire classification. The DeepRC ML method should be used in combination
     with the DeepRC encoder.
 
+    Important note: to be able to run DeepRC, you must have access to a GPU. DeepRC does not work on a CPU.
```

Comment:
"DeepRC uses PyTorch functionalities that depend on GPU.. " or sth like that? to show it's up to pytorch?